pubspec.yaml

```diff
@@ -11,27 +11,27 @@ environment:
   sdk: ">=2.11.0 <3.0.0"
 
 dependencies:
-  args: '>=1.5.1 <3.0.0'
-  glob: ^1.1.7
-  http: ^0.12.0
-  http_multi_server: ^2.1.0
-  io: '>=0.3.3 <2.0.0'
-  logging: '>=0.11.3+2 <2.0.0'
-  meta: ">=1.1.7 <1.7.0" # pin to avoid issue in 1.7.0
-  pedantic: ^1.7.0
-  pub_semver: '>=1.4.2 <3.0.0'
-  shelf: ^0.7.2
-  shelf_proxy: ^0.1.0
-  stack_trace: ^1.9.3
+  args: ^2.3.1
+  glob: ^2.0.1
+  http: ^0.13.3
+  http_multi_server: ^3.2.1
+  io: ^1.0.3
+  logging: ^1.1.0
+  meta: ^1.8.0
+  pedantic: ^1.11.1
+  pub_semver: ^2.1.2
+  shelf: ^1.2.0
+  shelf_proxy: ^1.0.0
+  stack_trace: ^1.10.0
```

Comment:
The regular dependency upgrades are safe because webdev_proxy is installed globally